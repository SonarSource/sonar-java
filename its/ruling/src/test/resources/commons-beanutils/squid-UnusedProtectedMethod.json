{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/WrapDynaBean.java':[
383,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/locale/LocaleBeanUtils.java':[
561,
580,
596,
619,
639,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/locale/LocaleConvertUtils.java':[
330,
346,
],
}
