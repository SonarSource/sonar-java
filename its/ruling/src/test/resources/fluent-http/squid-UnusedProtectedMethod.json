{
'net.code-story:http:src/main/java/net/codestory/http/filters/auth/CookieAuthFilter.java':[
190,
],
'net.code-story:http:src/main/java/net/codestory/http/injection/GuiceConfiguration.java':[
27,
],
}
