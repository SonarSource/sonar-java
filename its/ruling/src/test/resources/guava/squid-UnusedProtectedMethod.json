{
'com.google.guava:guava:src/com/google/common/base/Optional.java':[
227,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableList.java':[
350,
],
'com.google.guava:guava:src/com/google/common/collect/Iterators.java':[
1059,
],
'com.google.guava:guava:src/com/google/common/collect/Sets.java':[
0,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AbstractExecutionThreadService.java':[
45,
83,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AbstractIdleService.java':[
49,
64,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/MoreExecutors.java':[
0,
908,
],
}
