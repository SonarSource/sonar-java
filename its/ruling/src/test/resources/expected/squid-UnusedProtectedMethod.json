{
'project:com/sun/corba/se/impl/activation/ORBD.java':[
257,
297,
],
'project:com/sun/corba/se/impl/encoding/IDLJavaSerializationOutputStream.java':[
602,
],
'project:com/sun/corba/se/impl/io/IIOPInputStream.java':[
962,
],
'project:com/sun/corba/se/impl/legacy/connection/LegacyServerSocketManagerImpl.java':[
180,
],
'project:com/sun/corba/se/impl/orb/ORBSingleton.java':[
391,
395,
],
'project:com/sun/corba/se/impl/transport/SocketOrChannelConnectionImpl.java':[
1645,
],
'project:com/sun/imageio/plugins/jpeg/JPEGImageReader.java':[
262,
],
'project:com/sun/java/swing/plaf/motif/MotifBorders.java':[
285,
],
'project:com/sun/java/swing/plaf/motif/MotifButtonUI.java':[
99,
],
'project:com/sun/java/swing/plaf/motif/MotifComboBoxUI.java':[
0,
],
'project:com/sun/java/swing/plaf/motif/MotifDesktopIconUI.java':[
126,
],
'project:com/sun/java/swing/plaf/motif/MotifPopupMenuUI.java':[
104,
],
'project:com/sun/java/swing/plaf/motif/MotifScrollPaneUI.java':[
65,
],
'project:com/sun/java/swing/plaf/windows/WindowsFileChooserUI.java':[
0,
0,
0,
],
'project:com/sun/jmx/snmp/daemon/SnmpSubBulkRequestHandler.java':[
62,
],
'project:com/sun/jmx/snmp/daemon/SnmpSubNextRequestHandler.java':[
68,
],
'project:com/sun/jmx/snmp/daemon/SnmpSubRequestHandler.java':[
70,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FilterExpr.java':[
69,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TemplatesImpl.java':[
277,
284,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMConfigurationImpl.java':[
221,
],
'project:com/sun/org/apache/xerces/internal/dom/ElementNSImpl.java':[
201,
],
'project:com/sun/org/apache/xerces/internal/dom/NamedNodeMapImpl.java':[
537,
551,
],
'project:com/sun/org/apache/xerces/internal/impl/XML11DTDScannerImpl.java':[
222,
239,
289,
303,
],
'project:com/sun/org/apache/xerces/internal/impl/XML11DocumentScannerImpl.java':[
584,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentFragmentScannerImpl.java':[
1123,
1757,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityScanner.java':[
1923,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLStreamFilterImpl.java':[
82,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLStreamReaderImpl.java':[
1300,
1310,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/DTDGrammar.java':[
1827,
1831,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
1476,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/util/Base64.java':[
99,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
234,
238,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/Token.java':[
204,
207,
216,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
1830,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractDOMParser.java':[
319,
],
'project:com/sun/org/apache/xerces/internal/parsers/NonValidatingConfiguration.java':[
778,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
2305,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XPointerElementHandler.java':[
738,
758,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ExtendedType.java':[
74,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/IncrementalSAXSource_Filter.java':[
493,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2RTFDTM.java':[
201,
],
'project:com/sun/org/apache/xml/internal/serializer/ToHTMLStream.java':[
1788,
],
'project:com/sun/org/apache/xml/internal/serializer/ToXMLSAXHandler.java':[
599,
],
'project:com/sun/org/apache/xml/internal/serializer/ToXMLStream.java':[
545,
],
'project:com/sun/org/apache/xpath/internal/compiler/XPathParser.java':[
676,
1158,
1178,
1204,
2000,
],
'project:com/sun/org/apache/xpath/internal/jaxp/XPathExpressionImpl.java':[
72,
],
'project:com/sun/org/apache/xpath/internal/objects/XObject.java':[
684,
],
'project:java/awt/EventQueue.java':[
879,
],
'project:java/awt/MenuItem.java':[
437,
452,
],
'project:java/awt/MenuShortcut.java':[
201,
],
'project:java/awt/datatransfer/DataFlavor.java':[
1343,
1359,
],
'project:java/awt/dnd/DropTargetContext.java':[
137,
],
'project:java/awt/event/InvocationEvent.java':[
222,
],
'project:java/awt/font/TextLayout.java':[
837,
],
'project:java/awt/image/Raster.java':[
1024,
],
'project:java/beans/beancontext/BeanContextServicesSupport.java':[
992,
1071,
],
'project:java/beans/beancontext/BeanContextSupport.java':[
1196,
1210,
1224,
1329,
1340,
],
'project:java/io/ObjectOutputStream.java':[
725,
],
'project:java/io/PrintStream.java':[
422,
],
'project:java/io/PrintWriter.java':[
375,
388,
],
'project:java/lang/SecurityManager.java':[
402,
460,
486,
503,
],
'project:java/lang/reflect/Proxy.java':[
267,
],
'project:java/net/DatagramSocket.java':[
196,
],
'project:java/net/SocksSocketImpl.java':[
918,
],
'project:java/net/URL.java':[
661,
],
'project:java/rmi/server/UnicastRemoteObject.java':[
164,
],
'project:java/security/SecureRandom.java':[
225,
],
'project:java/util/Observable.java':[
173,
],
'project:java/util/concurrent/ForkJoinPool.java':[
1846,
],
'project:java/util/concurrent/Semaphore.java':[
648,
699,
],
'project:java/util/concurrent/locks/ReentrantLock.java':[
621,
681,
748,
],
'project:java/util/concurrent/locks/ReentrantReadWriteLock.java':[
1248,
1320,
1335,
1393,
1460,
],
'project:java/util/logging/Logger.java':[
251,
],
'project:javax/management/StandardEmitterMBean.java':[
185,
],
'project:javax/management/StandardMBean.java':[
230,
],
'project:javax/management/modelmbean/RequiredModelMBean.java':[
2744,
],
'project:javax/management/relation/RelationTypeSupport.java':[
195,
291,
],
'project:javax/sound/midi/MidiDevice.java':[
315,
],
'project:javax/sound/sampled/Mixer.java':[
282,
],
'project:javax/sound/sampled/ReverbType.java':[
185,
],
'project:javax/swing/DefaultDesktopManager.java':[
576,
586,
],
'project:javax/swing/JMenu.java':[
1087,
],
'project:javax/swing/JScrollBar.java':[
672,
],
'project:javax/swing/Popup.java':[
83,
],
'project:javax/swing/plaf/LayerUI.java':[
584,
],
'project:javax/swing/plaf/basic/BasicComboBoxUI.java':[
1090,
1156,
],
'project:javax/swing/plaf/basic/BasicComboPopup.java':[
0,
0,
0,
0,
0,
0,
0,
],
'project:javax/swing/plaf/basic/BasicDesktopPaneUI.java':[
0,
0,
0,
0,
0,
],
'project:javax/swing/plaf/basic/BasicDirectoryModel.java':[
204,
],
'project:javax/swing/plaf/basic/BasicFileChooserUI.java':[
0,
],
'project:javax/swing/plaf/basic/BasicInternalFrameTitlePane.java':[
369,
],
'project:javax/swing/plaf/basic/BasicInternalFrameUI.java':[
0,
0,
],
'project:javax/swing/plaf/basic/BasicListUI.java':[
611,
626,
1064,
1078,
1563,
],
'project:javax/swing/plaf/basic/BasicMenuItemUI.java':[
0,
],
'project:javax/swing/plaf/basic/BasicScrollPaneUI.java':[
0,
],
'project:javax/swing/plaf/basic/BasicSliderUI.java':[
781,
792,
803,
814,
],
'project:javax/swing/plaf/basic/BasicSplitPaneDivider.java':[
797,
],
'project:javax/swing/plaf/basic/BasicSplitPaneUI.java':[
542,
561,
580,
599,
618,
1265,
1599,
1616,
1643,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
2117,
],
'project:javax/swing/plaf/basic/BasicToolBarUI.java':[
0,
471,
],
'project:javax/swing/plaf/basic/BasicTreeUI.java':[
394,
461,
472,
1897,
1906,
],
'project:javax/swing/plaf/metal/MetalComboBoxUI.java':[
239,
300,
],
'project:javax/swing/plaf/metal/MetalFileChooserUI.java':[
0,
],
'project:javax/swing/plaf/metal/MetalRadioButtonUI.java':[
102,
],
'project:javax/swing/plaf/metal/MetalSliderUI.java':[
507,
],
'project:javax/swing/plaf/metal/MetalTabbedPaneUI.java':[
1163,
],
'project:javax/swing/plaf/metal/MetalToolBarUI.java':[
0,
0,
],
'project:javax/swing/plaf/metal/MetalTreeUI.java':[
144,
],
'project:javax/swing/plaf/nimbus/NimbusLookAndFeel.java':[
420,
],
'project:javax/swing/table/TableColumn.java':[
803,
],
'project:javax/swing/text/AsyncBoxView.java':[
187,
],
'project:javax/swing/text/FlowView.java':[
586,
],
'project:javax/swing/text/LabelView.java':[
168,
],
'project:javax/swing/text/PlainView.java':[
190,
],
'project:javax/swing/text/WrappedPlainView.java':[
217,
],
'project:javax/swing/text/ZoneView.java':[
188,
],
'project:javax/swing/text/html/AccessibleHTML.java':[
2860,
],
'project:javax/swing/text/html/HTMLDocument.java':[
2740,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
834,
],
'project:javax/swing/text/html/MinimalHTMLWriter.java':[
641,
],
'project:javax/swing/text/html/parser/DTD.java':[
287,
],
'project:javax/swing/text/html/parser/Parser.java':[
209,
],
'project:javax/swing/tree/FixedHeightLayoutCache.java':[
1202,
1427,
1446,
],
'project:javax/tools/ForwardingJavaFileManager.java':[
54,
],
'project:javax/tools/ForwardingJavaFileObject.java':[
49,
],
'project:javax/tools/SimpleJavaFileObject.java':[
63,
],
}
